src/CommunityPatch/Patches/Perks/Intelligence/Engineering/ImperialFirePatch.cs

```diff
@@ -75,22 +76,8 @@ public override void Reset()
     }
 
     public override void Apply(Game game) {
-      var textObjStrings = TextObject.ConvertToStringList(
-        new List<TextObject> {
-          _perk.Name,
-          _perk.Description
-        }
-      );
-      // most of the properties of skills have private setters, yet Initialize is public
-      _perk.Initialize(
-        textObjStrings[0],
-        textObjStrings[1],
-        _perk.Skill,
-        (int) _perk.RequiredSkillValue,
-        _perk.AlternativePerk,
-        _perk.PrimaryRole, 1f,
-        _perk.SecondaryRole, _perk.SecondaryBonus
-      );
+      _perk.Modify(1f, SkillEffect.EffectIncrementType.Add);
```

Comment:
If anyone's wondering, we pass `EffectIncrementType.Add` because it's left out of the call to `Initialize` and this is the default argument there.

Reply:
oh sweet!